exceptions/include/peach_library/exceptions/core/prefix_manager.hpp

```diff
@@ -0,0 +1,82 @@
+#ifndef PEACH_PREFIX_MANAGER_HPP
+#define PEACH_PREFIX_MANAGER_HPP
+
+#include <string>
+
+namespace peach
+{
+  enum class ExceptionTypes;
+} // namespace peach
+
+namespace peach
+{
+
+  // Short info:
+  // * Provides general way to set default prefix
+  // Link docu:
+  class PrefixManager
```

Comment:
Creating instances of this does not really make any sense. So either the constructor should be protected (for inheritance to work), or we replace the class with a namespace that stores the default prefix (preferred by me).